File: shoebill_plugins/PropertyPlugin/src/main/java/lt/ltrp/property/event/PlayerEnterPropertyEvent.java
package lt.ltrp.property.event;


import lt.ltrp.object.LtrpPlayer;
import lt.ltrp.property.object.Property;

/**
 * @author Bebras
 *         2015.11.29.
 */
public class PlayerEnterPropertyEvent extends PropertyEvent {

    private LtrpPlayer player;

    public PlayerEnterPropertyEvent(Property property, LtrpPlayer player) {
        super(property);
        this.player = player;
    }

    public LtrpPlayer getPlayer() {
        return player;
    }
}

File: shoebill_plugins/PropertyPlugin/src/main/java/lt/ltrp/property/event/PlayerEnterPropertyEvent.kt
package lt.ltrp.property.event


import lt.ltrp.`object`.LtrpPlayer
import lt.ltrp.property.`object`.Property

/**
 * @author Bebras
 * *         2015.11.29.
 */
class PlayerEnterPropertyEvent(property: Property, val player: LtrpPlayer) : PropertyEvent(property)
